### PR TITLE
Members only

### PR DESCRIPTION
**Pull request by AlejandroNo4**
This project was made for learning proposes, to implement authentication systems using Ruby On Rails, so users can only access areas of a site they are authorized to.

### CODE REVIEW

File: app/views/home/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    </div>  
+    <div class="card-body">
+      <p class="card-text">
+        <%= post.body %>
+      </p>
+      <% if signed_in? %>
+        <p class="card-text">
+          By: <%= post.user.email %>
+        </p>
+      <% end %>
```
Comment (lekegitrepo) [inline app/views/home/index.html.erb]: - Remove logic from the view

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            <%= link_to 'Sign out', destroy_user_session_path, method: :delete, class: 'nav-link' %>
+          </li>
+        <% else %>
+          <li class="nav-item">
+            <%= link_to 'Register', new_user_registration_path, class: 'nav-link' %>
+          </li>
+          <li class="nav-item">
+            <%= link_to 'Sign in', new_user_session_path, class: 'nav-link' %>
+          </li>
+        <% end %>
```
Comment (lekegitrepo) [inline app/views/shared/_navbar.html.erb]: - Remove logic from the view

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Changes Required! :recycle:
### Hi @AlejandroNo4 :clap: :clap:
#### You have done a good job :muscle: However, you need to implement some minor changes before this project can be approved. Check my comments below :point_down:

#### 1) Readme:
 - Add instruction on how to run the migration
 
#### 2) View & Logic:
 - There should be no logic in the view. Move any view logic to the helper.

#### _Reviewed by Temitayo Adeleke_

**Review by mke2111 — APPROVED** [review]
# STATUS: APPROVED ㊗️ 🟡 

## Hey Happy Coder, 👋🏽 
Congratulations, 🥂 . You got minimum requirements for approval.

### You may now merge to the develop branch and remember to submit a project completion form.
### I wish you the very best in your upcoming projects.

**Cheers 🍹 and Happy coding 💻** 

## Reach me on slack for a chit-chat 🆘  Roy Mukuye

**Pull request merged**